README.md

```diff
@@ -0,0 +1,105 @@
+# JavaScript-Capstone-Project (RestaurantMeal)
+
+> This project is build around the **MealDB API** using HTML5,SCSS and JavaScript. You can view various types of **Dishes** & **Meals** Moreover, you can interact with the meals list by giving them a **like** or by adding some comments about them.
+
+## Built With ⚙️
+
+- **Major languages**: HTML, CSS, & JavaScript.
+- **Frameworks**: No-Frameworks.
+- **Linters**: Lighthouse, Webhint, Stylelint and ESLint.
+- **Technologies used**: NPM, Webpack, & Jest Library.
+- **Tools**: Git, Gitflow, GitHub, VS Code, & Chrome Browser.
+
+## Live Demo (Deployed on GitHub-Pages with SSL)
+
+[GitHub page](https://graceabike.github.io/JavaScript-Capstone-Project/)
```

Comment:
- [ ] Kindly add `/dist/index.html` to this link to take the user directly to your page and not to the README file 👍